tests/accumulator_test.py

```diff
@@ -466,73 +466,6 @@ def test_with_noise(self):
                                delta=4)
 
 
-class SumAccumulatorTest(unittest.TestCase):
```

Comment:
This test started failed for some unclear reasons. Accumulator are already deprecated (will be removed next week), so it's easier to remove the test :)